http_service/requirements.txt

```diff
@@ -10,3 +10,4 @@ marshmallow==3.7.1
 requests==2.24.0
 rq==1.4.3
 rq-dashboard==0.6.1
+sentry-sdk==0.10.2
```

Comment:
Why not the latest version of sentry-sdk?

Reply:
When setting up a new project via IT's deployment of Sentry it showed that version. I've fixed it. Thanks for catching it!